harvard-university-for-the-creative-arts.csl

```diff
@@ -398,6 +458,13 @@
         </group>
         <text macro="legal-detail"/>
         <text macro="locator"/>
+        <group>
+          <choose>
+            <if type="map" match="any">
+              <text variable="collection-title" prefix="(" suffix=")"/>
+            </if>
+          </choose>
+        </group>
```

Comment:
I'm not too sure what I'm doing with this part, if I remove line 467 it won't validate. Do I need to remove more than that one line?